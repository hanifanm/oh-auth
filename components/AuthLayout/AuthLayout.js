import styles from './AuthLayout.scss'
import React from 'react'
import cx from 'classnames'

import logo from './logo1.png'

const AuthLayout = ({ className, children }) => {
  return (
    <div className={cx(className, styles.root)}>
      <div className={styles.screen}>
        <div className={styles.authContainer}>
          <img src={logo} className={styles.logo} />
          {children}
        </div>
      </div>
    </div>
  );
}

export default AuthLayout;