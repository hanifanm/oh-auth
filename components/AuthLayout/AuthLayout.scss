.root {
  min-height: 100vh;
  padding: 80px 0 100px 0;
}

.authContainer {
  max-width: 340px;
  padding: 0 20px;
  width: 100%;
  margin: 0 auto;
}

.logo {
  height: 80px;
  display: block;
  margin: 0 auto 20px auto;
}
