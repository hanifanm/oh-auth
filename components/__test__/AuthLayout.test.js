import { configure, shallow } from 'enzyme'
import React from 'react'
import sinon from 'sinon'
import Adapter from 'enzyme-adapter-react-16'
import AuthLayout from '../AuthLayout/AuthLayout'

configure({ adapter: new Adapter() })

describe('<AuthLayout />', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<AuthLayout />)
    expect(wrapper).toMatchSnapshot()
  })
})