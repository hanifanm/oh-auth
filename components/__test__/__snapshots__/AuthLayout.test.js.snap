// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AuthLayout /> matches snapshot 1`] = `
<div
  className=""
>
  <div>
    <div>
      <img
        src={
          Object {
            "canInstrument": true,
            "createTransformer": [Function],
            "getCacheKey": [Function],
            "process": [Function],
          }
        }
      />
    </div>
  </div>
</div>
`;
